mmv1/products/cloudids/api.yaml

```diff
@@ -46,9 +46,10 @@ objects:
     name: 'Endpoint'
     base_url: 'projects/{{project}}/locations/{{location}}/endpoints'
     create_url: 'projects/{{project}}/locations/{{location}}/endpoints?endpointId={{name}}'
+    update_url: 'projects/{{project}}/locations/{{location}}/endpoints/{{name}}?update_mask=threat_exceptions'
```

Comment:
We can use  `update_mask: true` instead of hard coding update_url

https://github.com/GoogleCloudPlatform/magic-modules/blob/59946b96064bcf8db713ce4857b47e7448ccc87b/mmv1/api/resource.rb#L50

Reply:
Sounds good, I'll add it.